apps/web/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "web",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fluentui/react-native": "^0.15.55",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "react": "^16.13.1",
+    "react-art": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-native": "0.62.2",
+    "react-native-web": "^0.12.3",
+    "react-scripts": "3.4.1",
+    "typescript": "~3.7.2",
```

Comment:
You should see if you can line up the versions for supporting packages like typescript with the ones already in the repo. Look in /scripts/package.json for the canonical versions of many of these. Right now it looks like we have typescript: "3.8.3" and "webpack": "^4.29.5". The webpack version should resolve to match 4.42.0 (which you have here) but I would suggest updating the one in scripts to match this version.

So try changing this TS version to match the version in scripts, then change scripts webpack to match this one.